gallery/src/styles/PageLayout/PageLayoutPage.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
+import RawHtmlExample from '../../CodeExample';
+
+const SingleColumnLayoutExampleCode = require('!!raw-loader!./SingleColumnLayoutExample.html').default,
+    TwoColumnLayoutExampleCode = require('!!raw-loader!./TwoColumnLayoutExample.html').default,
+    ReactRootLayoutExampleCode = require('!!raw-loader!./ReactRootLayoutExample.html').default;
```

Comment:
These variables should all start with lowercase letters

Reply:
> These variables should all start with lowercase letters

https://github.com/sonatype/sonatype-react-shared-components/pull/28/commits/97efbdff0dea563d0e39985933b6dfd795b29104